tests/benchmarks/group_a/betweenness-10hop.yml

```
@@ -0,0 +1,22 @@
+    - ["GRAPH.QUERY", "graph", "CREATE (s0:S0) WITH * UNWIND range(1, 1000) AS x CREATE (:S)-[:R]->(s0)-[:R]->(:S)"]
+    - ["GRAPH.QUERY", "graph", "CREATE (s1:S1) WITH * UNWIND range(1, 1000) AS x CREATE (:S)-[:R]->(s1)"]
+    - ["GRAPH.QUERY", "graph", "MATCH (s0:S0), (s1:S1) WITH * CREATE (s1)-[:R]->(s0)"]
+
+
+parameters:
+  num_clients: 32
+  num_requests: 12800
+  random_int_max: 262016
+  random_seed: 12345
+  queries:
+    - query: 'CYPHER Id1=__rand_int__ CALL algo.betweenness({samplingSeed: $Id1, 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)